server/routes/login.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+
+router.post('/api/login', (req, res) => {
+  const { email, password } = req.body;
+  // Here we would typically check the credentials against a database
```

Comment:
This implementation is not secure and should not be used in a production environment. Always validate user credentials against a secure data store. Never accept any credentials.